pages/terms.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page-no-toc
+title: "Terms of Use"
+permalink: terms
+---
+
+# Terms of Use
+
+## URI Persistence Policy
+
+Last modified: 2021-01-12
```

Comment:
There seems to exist a plugin to handle this automatically: https://github.com/gjtorikian/jekyll-last-modified-at (there may be more, that's just the first result). Do you think it's worth looking into it not to update this manually ?

Reply:
Unfortunately GitHub limits us to a select number of plugins that does not include that one: https://pages.github.com/versions/

Reply:
This might sound strange but I would've suggested to leave this manual. Significant content changes on this document (eg. adding a URI to the list) is not expected to change often. If and when it changes, it should be intentional.